### PR TITLE
Redesign Event Modal

### LINKED ISSUE #127: EventModal Improvements
image

- [x] Remove the add to calendar buttons for past events (and currently happening?)
- [x] Fix whitespace wrapping
- [x] Indicate attendance in the modal and prompt to give feedback if they haven't already submitted anything for an attended event

### LINKED ISSUE #173: better mobile event modal
sorry I'm a hater but the event modal is so extremely narrow on mobile and looks weird with this much wrapping imo - wondering if we can media query and reduce padding or horizontally fill the page somehow that looks nicer. Other option is shrink all font sizes on mobile

image

also I don't know what purpose there is to link to the event on the main website to gather the same information and arguably less usability once we have feedback functionality. I don't think the plain text link looks very good so maybe remove

### PR DESCRIPTION
**Pull request by alexzhang1618**
# Info

Closes #173 
Closes #127 

## Changes

- Update EventModal design
- Switching to a grid layout, moving EventBadges into its own component

# Type of Change

- [ ] Bug Fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
      expected)
- [ ] Logistics Change (A change to a README, description, or dev workflow setup like
      linting/formatting)
- [ ] Continuous Integration Change (Related to deployment steps or continuous integration
      workflows)
- [ ] Other: (Fill In) 

# Testing

I have tested that my changes fully resolve the linked issue ...

- [x] locally on Desktop.
- [x] on the live deployment preview on Desktop.
- [x] on the live deployment preview on Mobile.
- [x] I have added new Cypress tests that are passing.

# Checklist

- [x] I have performed a self-review of my own code.
- [x] I have followed the style guidelines of this project.
- [x] I have documented any new functions in `/src/lib/*` and commented hard to understand areas
      anywhere else.
- [x] My changes produce no new warnings.

# Screenshots
Desktop
Screenshot 2024-03-29 at 3 51 22 PM
Tablet
Screenshot 2024-03-29 at 3 51 43 PM
Mobile
Screenshot 2024-03-29 at 3 52 08 PM

### CODE REVIEW

File: src/components/events/EventDetail/index.tsx [hunk 1]
```diff
@@ -1,12 +1,10 @@
-import { CommunityLogo, Typography } from '@/components/common';
+import { Typography } from '@/components/common';
 import CalendarButtons from '@/components/events/CalendarButtons';
-import PointsDisplay from '@/components/events/PointsDisplay';
```
Comment (SheepTester) [inline src/components/events/EventDetail/index.tsx:3]: `CommunityLogo` and `PointsDisplay` aren't being used anywhere now, so you should retire them 🫡

**Review by SheepTester — CHANGES_REQUESTED** [review]
image

i dont see the add to calendar buttons anymore

**Comment by alexzhang1618** [comment]
> i dont see the add to calendar buttons anymore

It's because live events aren't considered upcoming, so the calendar buttons are removed. Should we keep them if the event is live?

**Comment by SheepTester** [comment]
why are the image sizes different

image


image

**Comment by alexzhang1618** [comment]
> why are the image sizes different

erm I was playing around with grid sizes to see what fits best and I have a 1fr thingy with a minmax of 20 or so rem on the image area. should it be like 50vw or something else?

**Comment by SheepTester** [comment]
I think you should keep the right side at least 30rem so the "add to calendar" buttons don't wrap. `grid-template-columns: 1fr 30rem;` or `grid-template-columns: 1fr minmax(30rem, 1fr);` both seem fine

image

image

I would also recommend adding `white-space: pre-wrap;` for the description to support line breaks

**Review by SheepTester — APPROVED** [review]
looks good! 🎉

**Pull request merged**